### ISSUE TITLE
LocalDate.ToString does not provide calendar information

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nodatime/nodatime#23]
```
It would be nice if "round trip" for LocalDate{,Time} parsing really was 
completely round-trip, including calendar - at least for a known set of 
calendars. Currently although formatting and parsing *use* the calendar 
information, it's not part of the textual representation.

We should possibly have a format specifier for "calendar name". It would *at 
least* help when debugging.
```

Original issue reported on code.google.com by `jonathan.skeet` on 15 Oct 2011 at 10:24
- Blocked on: #138
